test/lib/y2caasp/cfa/systemd_timesyncd_test.rb

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env rspec
+
+require_relative "../../../test_helper"
+require "y2caasp/cfa/systemd_timesyncd"
+require "cfa/memory_file"
+
+
+def timesyncd_disk_content
+  path = File.expand_path("../../../../fixtures/cfa/timesyncd", __FILE__)
```

Comment:
You would like to use [FIXTURES_PATH](https://github.com/yast/yast-caasp/blob/master/test/test_helper.rb#L59) instead.